super-mode-calculator/src/parse.ts

```diff
@@ -1,13 +1,6 @@
 import type { SimpleQueryRowsResponse } from '@google-cloud/bigquery';
-import type { GetQueryResultsOutput } from 'aws-sdk/clients/athena';
 import { z } from 'zod';
-import { QueryReturnedInvalidDataError } from '../../lib/errors';
-
-const URL_COL = 0;
-const REGION_COL = 1;
-const TOTAL_AV_COL = 2;
-const TOTAL_VIEWS_COL = 3;
-const AV_PER_VIEW_COL = 4;
+import { QueryReturnedInvalidDataError } from './lib/errors';
```

Comment:
this is the only error type that's being used, so the others in `errors.ts` can be deleted